### PR TITLE
En 1470 data models edit

### PR DESCRIPTION
**Pull request by nicpuddu**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: test/ui/data-models/edit/EditDataModelPage.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { shallow } from 'enzyme';
+import EditDataModelPage from 'ui/data-models/edit/EditDataModelPage';
+
+describe('EditDataModelPage', () => {
+  let component;
+  beforeEach(() => {
+    component = shallow(<EditDataModelPage />);
+  });
+  it('renders without crashing', () => {
+    expect(component.exists()).toEqual(true);
```
Comment (fedelocci) [inline test/ui/data-models/edit/EditDataModelPage.test.js]: toBe

File: test/ui/data-models/edit/EditDataModelPage.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  });
+
+  it('verify if exist InternalPage with class EditDataModelPage', () => {
+    expect(component.find('InternalPage').hasClass('EditDataModelPage')).toEqual(true);
```
Comment (fedelocci) [inline test/ui/data-models/edit/EditDataModelPage.test.js]: toBe

File: src/ui/data-models/list/DataModelListTableContainer.js [hunk 3]
```diff
@@ -20,6 +23,9 @@ export const mapDispatchToProps = dispatch => ({
   onWillMount: (page) => {
     dispatch(fetchDataModelListPaged(page));
   },
+  onClickEdit: (dataModelId) => {
+    gotoRoute(ROUTE_DATA_MODEL_EDIT, { dataModelId });
```
Comment (mcarta-rt) [inline src/ui/data-models/list/DataModelListTableContainer.js]: You could use gotoRoute or into thunks either set a route for a Link component

Comment (nicpuddu) [reply src/ui/data-models/list/DataModelListTableContainer.js]: this is injected at the moment

**Comment by entando-service** [comment]
All is well

**Pull request merged**